### PR TITLE
Update tdot.md

### PR DESCRIPTION
**Pull request by shengda**
https://github.com/w3f/Grants-Program/pull/872

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

We propose amendment to tDOT grant because we enhance our original architecture design.
* During the implementation, we find it more robust and flexible to introduce the concept of local LP which is generated by individual stable asset pool and aggregate LP which is generated by local LP. tDOT is an aggregate LP. Therefore, we add a milestone for implement stable asset XCM pallet to manages both LPs;
* Minting and redeeming tDOT on host chains requires changes to the stable asset pallet due to change in milestone 1. Therefore, we expand the scope of milestone 2 and 3;
* We remove cross-chain swap from the grant since a cross-chain swap can be seen as a combination of cross-chain mint + redeem. We are going to implement such swap in DApp instead of on pallet;
* We remove cross-chain fee and yield handling from the grant since we have implemented an off-chain fee and yield handing solution. Since this solution is offchain and we have run it successfully on Kusama and Polkadot, we decide to remove them from the grant.

### CODE REVIEW

**Comment by cruikshankss** [comment]
Good day @shengda, thank you for your application. I will look into it as soon as possible.

File: applications/tdot.md [hunk 1]
```diff
@@ -101,68 +101,71 @@ NUTS Finance is a blockchain development DAO. Our team is composed of experience
 - **Full-Time Equivalent (FTE):**  2
 - **Total Costs:** 16,000 DAI
```
Comment (semuelle) [inline applications/tdot.md:102]: If I'm counting correctly, this should now be

```suggestion
- **Total Costs:** 14,000 DAI
```

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the amendment, @shengda. It sounds as if you are now planning to implement everything in one pallet, instead of several. Is that correct?

**Review by Noc2 — CHANGES_REQUESTED** [review]
I’m happy to approve it, once the total costs are updated.

**Comment by shengda** [comment]
> 

We will implement it in two pallets.
* Stable asset pallet is the upgraded version from our previous grant. It will be deployed on each parachain that supports stable asset and mints local LP;
* Stable asset XCM pallet manages all cross-chain minting and redemption. It will be deployed on the host chain of the aggregate LP.

**Review by semuelle — APPROVED** [review]
Thanks for the update. Happy to proceed.

**Review by Noc2 — APPROVED** [review]

**Review by alxs — APPROVED** [review]

**Pull request merged**